File: BLINK/blink.py
import RPi.GPIO as GPIO
import time
pin=40 

GPIO.setwarnings(False)
GPIO.setmode(GPIO.BOARD)
GPIO.setup(pin, GPIO.OUT)
GPIO.output(pin, GPIO.LOW)
a=0
while True:
	GPIO.output(pin, GPIO.HIGH)
	time.sleep(0.5)
	GPIO.output(pin, GPIO.LOW)
	time.sleep(0.5)
GPIO.cleanup()

File: BLINK/blink.cpp
#include <iostream>
#include <wiringPi.h>
using namespace std;
int ledPin=29;
void setup(){
	pinMode(ledPin,OUTPUT);
}

void loop(){
	digitalWrite(ledPin,HIGH);
	delay(500);
	digitalWrite(ledPin,LOW);
	delay(500);
}
int main(void){
	if(wiringPiSetup()<0){
		cout<<"setup wiring pi failed"<<endl;
		return 1;
	}
	setup();
	while(1){
		loop();
	}
	return 0;
}
